tensorflow_recommenders_addons/dynamic_embedding/python/kernel_tests/horovod_sync_train_test.py

```diff
@@ -49,6 +49,19 @@ def test_adam_minimize_trainable(self):
     self.common_minimize_trainable(base_opt, test_opt, name="adam")
 
   def common_minimize_trainable(self, base_opt, test_opt, name):
+    if (is_macos() and is_arm64()):
+      self.skipTest(
+          "Apple silicon devices don't support synchronous training based on Horovod."
+      )
+    from tensorflow.python.framework.errors_impl import NotFoundError
+
+    # TODO(rhdong): Recover the testing, if the horovod import error is fixed on macOS+TF2.7+.
```

Comment:
rhdong?

Reply:
> rhdong?


--It's OK to assign it to me. :)

Reply:
This line was part of a previous fixing patch that was merged into the master branch.